examples/example-kotlin/build.gradle

```diff
@@ -28,7 +28,7 @@ def grpcVersion = '1.18.0-SNAPSHOT' // CURRENT_GRPC_VERSION
 dependencies {
     def kotlinVerion = plugins.findPlugin("org.jetbrains.kotlin.jvm").kotlinPluginVersion
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${kotlinVerion}"
-    implementation "com.google.api.grpc:proto-google-common-protos:1.0.0"
+    implementation "com.google.api.grpc:proto-google-common-protos:1.12.0"
```

Comment:
We no longer need this now as `implementation` provides transitive dependency.

```
+--- io.grpc:grpc-protobuf:1.18.0-SNAPSHOT
|    +--- io.grpc:grpc-core:1.18.0-SNAPSHOT (*)
|    +--- com.google.protobuf:protobuf-java:3.5.1
|    +--- com.google.guava:guava:26.0-android (*)
|    +--- com.google.api.grpc:proto-google-common-protos:1.0.0 (*)
|    \--- io.grpc:grpc-protobuf-lite:1.18.0-SNAPSHOT
|         +--- io.grpc:grpc-core:1.18.0-SNAPSHOT (*)
|         \--- com.google.guava:guava:26.0-android (*)

```

Reply:
Done.